packages/app-project/.babelrc

```diff
@@ -2,6 +2,7 @@
   "env": {
     "test": {
       "plugins": [
+        "babel-plugin-styled-components",
```

Comment:
This plugin was previously used only in lib-content and lib-react-components. The plugin docs are [here](https://styled-components.com/docs/tooling#babel-plugin), but it helps with creating human-readable classnames for components, server side rendering, and transpiling template literals. I added it to all apps and libraries in FEM.